sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/GroupIntoBatches.java

```diff
@@ -48,25 +48,24 @@
  * are output to the output {@link PCollection}.
  *
  * <p>Windows are preserved (batches contain elements from the same window). Batches may contain
- * elements from more than one bundle
+ * elements from more than one bundle.
  *
- * <p>Example (batch call a webservice and get return codes)
+ * <p>Example (batch call a webservice and get return codes):
  *
  * <pre>{@code
```

Comment:
The combination of `<pre>` for preserving line breaks and `{@code` for preserving angle brackets and curly brackets _mostly_ works.

`@` is the problematic character. I see `{@literal @}` show up in many classes' Javadoc, and that method doesn't do what's intended. In current Java 8 builds, it looks like `@` inside `{@code}` is not a problem in the general case and doesn't need to be escaped. The one exception is when `@` is the first non-whitespace character in a line, in which case it appears that Javadoc thinks it's a tag like `@return` or `@param` and throws an error.

I propose a solution for that case below.